rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtilsTest.java

```diff
@@ -822,6 +834,10 @@ else if (CovidConstants.FormFields.PATIENT_INFO_DOB.equals(field.getString(JsonF
                 numOfPopulatedFields++;
             }
         }
+        else if (CovidConstants.FormFields.SAMPLER_NAME.equals(field.getString(JsonFormConstants.KEY))) {
+            assertEquals(field.getString(VALUE), ANM_PREFERRED_NAME);
+            numOfPopulatedFields++;
+        }
```

Comment:
We'll need a test for the sample shipment field as well.

Reply:
fixed.